src/components/Table/GridTable.tsx

```diff
@@ -1109,6 +1120,8 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
           ...(!isHeader && !!style.levels && style.levels[level]?.cellCss),
           // The specific cell's css (if any from GridCellContent)
           ...rowStyleCellCss,
+          // Apply active row styling for non-nested card styles.
+          ...(style.nestedCards === undefined && isActive ? Css.bgLightBlue50.$ : {}),
```

Comment:
Maybe use a `GridStyle.activeBgColor` just to pre-emptively make this style-aware/changeable?